src/include/types.h

```diff
@@ -250,6 +254,27 @@ inline std::ostream& operator<<(std::ostream& out, const std::multimap<A,B,Comp,
 
 } // namespace std
 
+
+
+template<class A, class B, class Comp, class Alloc>
+struct fmt::formatter<std::map<A,B,Comp,Alloc>> {
+  constexpr auto parse(format_parse_context& ctx)
+  {
+    return ctx.begin();
+  }
+
+  template <typename FormatContext>
+  auto format(std::map<A,B,Comp,Alloc> const& m, FormatContext& ctx)
```

Comment:
could you please put
```c++
const std::map<A, B, Comp, Alloc>& m
```
instead?

Reply:
a clear saying on east/west const debate :-) ?
OK